### PR TITLE
Map tc 1, 2, 5, and 6 back to pg 0

### PR DESCRIPTION
**Pull request by wendani**
Lossy traffic does not need to be mapped to different ingress PGs. They can all share the same ingress PG.

Signed-off-by: Wenda Ni <wenni@microsoft.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: files/build_templates/qos_config.j2 [hunk 1]
```diff
@@ -39,11 +39,11 @@
...
             "1": "1",
-            "2": "2",
+            "2": "1",
             "3": "3",
             "4": "4",
-            "5": "5",
-            "6": "6",
+            "5": "1",
+            "6": "1",
             "7": "7"
```
Comment (lguohan) [inline files/build_templates/qos_config.j2]: why don't we map all of them to zero?

Comment (wendani) [reply files/build_templates/qos_config.j2]: Mapped all lossy to zero

**Comment by lguohan** [comment]
@wendani , please also change your pr title.

**Comment by lguohan** [comment]
please also state the reason why you make this change?

**Pull request merged**